packages/taxdat/R/config_helpers.R

```diff
@@ -7,249 +7,101 @@
 #' @param covariate_names character vector with names of covariates to include in the model
 #' @param countries_names character vector with country codes to include in the model
 #' @param countries_ids character vector with country location ids to include in the model
-#'
+#' @param config_fname the name of the config file that will be generated or checked and updated 
+#' @param check_existing_config whether just check an existing config file or make a new one 
 #' @return string with yaml config file 
 #' @export
-automate_mapping_config <- function(p, covariate_names, countries_names, countries_ids){
+automate_mapping_config <- function(p, covariate_names, countries_names, countries_ids, config_fname, check_existing_config){
```

Comment:
We should use this function to add explicit file names to config: add_explicit_file_names_to_config (in file_name_functions)